### ISSUE TITLE
twilio@5.9.0: hash check failed

### ISSUE THREAD

**Issue by drarsalan** [twilio/scoop-twilio-cli#105]
Getting this while trying to install twilio CLI..

D:\Projects\twilio-conversations-demo-react [main ≡ +1 ~1 -1 ~]> scoop bucket add twilio-scoop https://github.com/twilio/scoop-twilio-cli
Checking repo... OK
The twilio-scoop bucket was added successfully.
D:\Projects\twilio-conversations-demo-react [main ≡ +1 ~1 -1 ~]> scoop install twilio
WARN  Purging previous failed installation of twilio.
ERROR 'twilio' isn't installed correctly.
Removing older version (5.9.0).
'twilio' was uninstalled.
Installing 'twilio' (5.9.0) [64bit] from twilio-scoop bucket
twilio-win32-x64.tar.xz (40.7 MB) [=======================================================] 100%
Checking hash of twilio-win32-x64.tar.xz ... ERROR Hash check failed!
App:         twilio-scoop/twilio
URL:         https://twilio-cli-prod.s3.amazonaws.com/twilio-win32-x64.tar.xz
First bytes: FD 37 7A 58 5A 00 00 04
Expected:    450092d7a7d9a7aeb9e5cd4c34ea172cea6115612633e1e976033c809932964d
Actual:      ede679aab2d7736815c35e78482a06a9aa5203e71da01a049e8f1b8460d40e8b

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/twilio/scoop-twilio-cli/issues/new?title=twilio%405.9.0%3a+hash+check+failed

**Comment by broadkent** [comment]
I'm getting the exact same issue. Running PowerShell as admin.
Not sure if I should post it as a separate issue to get more visibility. If there is no response here in the next day or so I will.

**Comment by PlatinumAutoArmor** [comment]
Same issue

**Comment by shrutiburman** [comment]
Hi, please re-try. We've updated the hash. 
To unblock you for now, you can also download the exe file from our [release page](https://github.com/twilio/twilio-cli/releases/tag/5.10.0).

**Comment by AbdulmalikAlayande** [comment]
@shrutiburman 
please am still having this!!!
"""
PS C:\Windows\System32> scoop install twilio-cli
WARN  Purging previous failed installation of twilio-cli.
ERROR 'twilio-cli' isn't installed correctly.
Removing older version (5.11.0).
'twilio-cli' was uninstalled.
Installing 'twilio-cli' (5.11.0) [64bit] from main bucket
twilio-v5.11.0-win32-x64.tar.xz (40.8 MB) [===================================================================================================================] 100%
Checking hash of twilio-v5.11.0-win32-x64.tar.xz ... ok.
Extracting twilio-v5.11.0-win32-x64.tar.xz ... ERROR Exit code was 2!
Failed to extract files from C:\Users\USER\scoop\apps\twilio-cli\5.11.0\twilio-v5.11.0-win32-x64.tar.
Log file:
  ~\scoop\apps\twilio-cli\5.11.0\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=twilio-cli%405.11.0%3a+decompress+error
"""

**Comment by tiwarishubham635** [comment]
Fixed by https://github.com/twilio/twilio-cli/pull/658
